web3/scripts/release/test_wheel_install.sh

```diff
@@ -0,0 +1,7 @@
+rm -rf build dist
```

Comment:
Recommend using #!/bin/bash for the script. Can also use `set -e` to exit on error if that's useful.

Not sure what the equivalent is for windows, maybe it also follows some best practices.